src/components/feedback/Feedback.tsx

```diff
@@ -1,43 +1,62 @@
 import React, { FC, useReducer, ChangeEvent, useMemo, useCallback, MouseEvent } from 'react';
 
-import commonStyle from '../../styles/commonStyles.css';
+import commonStyle from 'styles/commonStyles.css';
 import {
   feedbackReducer,
   CHANGE_FEEDBACK_STATE,
-  RESET_STATE
-} from '../../reducers/feedbackReducer';
+  RESET_STATE,
+  FeedbackState,
+  FeedbackActionData
+} from 'reducers/feedbackReducer';
 import styles from './Feedback.css';
 import useValidate from 'hooks/useValidate';
 import { setInLocalStorage, FEEDBACK_DATA_LS_LABEL } from 'services/localStorage';
+import { ValidateElementInfoType } from 'hooks/useValidate';
+import { getBindedStyles } from 'helpers';
+
+const getClasses = getBindedStyles(styles);
 
 export const Feedback: FC = () => {
-  const [state, dispatch] = useReducer(feedbackReducer, {
+  const [state, dispatch] = useReducer<
+    (state: FeedbackState, { type, payload }: FeedbackActionData) => FeedbackState
+  >(feedbackReducer, {
```

Comment:
you can use built-in type `Reducer` like here:
https://codesandbox.io/s/react-typescript-hooks-2-y7zvkl?file=/src/App/App.tsx:441-448